client/src/components/productDetail/Top/ImageSlider/Thumbnails.jsx

```diff
@@ -1,69 +1,50 @@
 import React from 'react';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
 
-const Img1 = styled.img`
-border: 1px solid grey;
+const ThumbnailsDiv = styled.div`
+  border: 1px solid grey;
   position: absolute;
-  top: 10px;
   left: 10px;
   width:60px;
   height:60px;
   overflow:hidden;
 `;
 
+const Img1 = styled.img`
+  top: 10px;
+`;
+
 const Img2 = styled.img`
-  border: 1px solid grey;
-  position: absolute;
   top: 80px;
-  left: 10px;
-  width:60px;
-  height:60px;
-  overflow:hidden;
 `;
 
 const Img3 = styled.img`
-  border: 1px solid grey;
-  position: absolute;
   top: 150px;
-  left: 10px;
-  width:60px;
-  height:60px;
-  overflow:hidden;
 `;
 
 const Img4 = styled.img`
-  border: 1px solid grey;
-  position: absolute;
   top: 230px;
-  left: 10px;
-  width:60px;
-  height:60px;
-  overflow:hidden;
 `;
 
 const Img5 = styled.img`
-  border: 1px solid grey;
-  position: absolute;
   top: 300px;
-  left: 10px;
-  width:60px;
-  height:60px;
-
-  overflow:hidden;
 `;
 
-function Thumbnails({ images, currentIndex }) {
-  console.log(images, currentIndex);
-
+function Thumbnails({ photo, currentIndex }) {
   return (
-    <div>
-      <Img1 src={images[0].img1} />
-      <Img2 src={images[1].img2} />
-      <Img3 src={images[2].img3} />
-      <Img4 src={images[3].img4} />
-      <Img5 src={images[4].img5} />
-    </div>
+    <ThumbnailsDiv>
+      <Img1 />
+      <Img2 />
+      <Img3 />
+      <Img4 />
+      <Img5 />
```

Comment:
Maybe this can be further refactored one day using a map function and passing in 'top' as a prop in case the Img count grows.